src/views/Farms/components/FarmCard/ApyButton.tsx

```diff
@@ -25,6 +25,7 @@ export interface ApyButtonProps {
   lpRewardsApr?: number
   addLiquidityUrl?: string
   strikethrough?: boolean
+  useTooltipText?: boolean
```

Comment:
maybe you can use 
`const { isMobile } = useMatchBreakpointsContext()` 
to control should show we the tooltip details for mobile

Reply:
okay, wait a sec.

Reply:
IsMobile cannot achieve logic, so it is recommended to use this version.